src/msg/async/AsyncConnection.h

```diff
@@ -307,7 +304,7 @@ class AsyncConnection : public Connection {
   atomic64_t ack_left, in_seq;
   int state;
   int state_after_send;
-  int sd;
+  ConnectedSocket cs;
```

Comment:
hmm, I hope we can follow this now. like "sd", I think it's a frequent variable, so short name should be ok